x-pack/plugins/apm/public/components/shared/truncate_with_tooltip/index.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiToolTip } from '@elastic/eui';
+import React from 'react';
+import styled from 'styled-components';
+import { truncate } from '../../../style/variables';
+
+const TooltipWrapper = styled.div`
+  width: 100%;
+  .euiToolTipAnchor {
```

Comment:
EUI team advise against it, if the CSS name changes this wouldn't work anymore. You could use styled_component and set the `anchorClassName` to the `EuiToolTip`.  Here is an example: https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/components/app/service_inventory/ServiceList/index.tsx#L47-L56